From: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Date: Wed, 15 May 2024 20:05:49 -0400
Subject: Ship dutch translation (po/nl.po)

8a9d4b55b09d04482b46055f0a60f01b86738df3 imported the nl.po file from
the 2.0 branch, but it is not being shipped in any of the tarballs
built as a result of make dist, even though all the other po/*.po
files are shipped.

This has already been corrected in 2.4, but was ignored for 2.2.

Gnupg-Bug-Id: T7120

---
 po/LINGUAS | 1 +
 1 file changed, 1 insertion(+)

diff --git a/po/LINGUAS b/po/LINGUAS
index cc9cf27ab..5e26a2c3b 100644
--- a/po/LINGUAS
+++ b/po/LINGUAS
@@ -16,6 +16,7 @@ id
 it
 ja
 nb
+nl
 pl
 pt
 ro
-- 
2.43.0

